content/en/docs/contribute/suggesting-improvements.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Suggesting content improvements
+slug: suggest-improvements
+content_template: templates/concept
+weight: 10
+card:
+  name: contribute
+  weight: 20
+---
+
+{{% capture overview %}}
+
+The easiest way to contribute to Kubernetes documentation file is to an issue about existing content,
+or a request for new content. All you need is a [GitHub account](https://github.com/join) and a web browser.
+
+All new work on Kubernetes documentation begins with an issue in GitHub. Kubernetes contributors
+then review, categorize and tag issues as needed. You or another member
```

Comment:
nit: This may be the ideal, but is not always the case. Sometimes, contributors present their ideas on the Slack channel or in a meeting.